src/atoms/image/image.tsx

```diff
@@ -55,13 +56,30 @@ export const BaseImage = ({
       imgAttrs.height = imgAspectRatio[1];
       break;
   }
-  return lightbox ? (
+  return (
     <Fragment>
-      <LightboxGallery />
-      <amp-img {...imgAttrs} lightbox={lightbox} />
+      <Head>
+        <style>{`
+          .hero-image img{
+            object-fit: contain;
+          }
+        `}</style>
+      </Head>
+      {lightbox ? (
+        <Fragment>
+          <LightboxGallery />
+          {layout === "responsive" ? (
```

Comment:
Keeping  only **responsive**  layout as others are not used , 
so removing   fixed / intrinsic / fixed-height layouts and **refactoring** accordingly

@ags1773